src/components/tools/Charts/LineArea/Chart.svelte

```diff
@@ -50,8 +55,8 @@
     noData = true;
     xmin = new Date(Date.UTC(1950, 0, 1));
     xmax = new Date(Date.UTC(2099, 0, 1));
-    ymin = 10;
-    ymax = 50;
+    ymin = 0;
```

Comment:
the values of `ymin` and ymax` dont' matter because we are not showing the yaxis labels. the `ymin` and `ymax` are used to show y axis lines in the empty chart